src/plugin.ts

```diff
@@ -31,7 +31,6 @@ import type {
   OpenBrowserOptions,
   OpenBrowserReturnType,
 } from './api';
-import { ALL_AUTH_FLOW_TYPES } from './api';
```

Comment:
Yeah, I think this does make sense to export as a public constant so that it’s easier to say e.g. “all flows except one specific one” if we add support for other ones in the future